Packs/TrendMicroApex/Integrations/TrendMicroApex/TrendMicroApex.yml

```diff
@@ -23,9 +23,14 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Fetch incidents
```

Comment:
didn't we say we are removing this until they respond?

Reply:
Yes, removing